files/KoreBuild/scripts/KoreBuild.psm1

```diff
@@ -492,8 +480,14 @@ function Get-MSBuildPath() {
     }
 
     Write-Verbose "vswhere = $vswherePath $vswhereArgs"
-
-    $installations = & $vswherePath @vswhereArgs | Out-String | ConvertFrom-Json
+    $vswhereOut = & $vswherePath @vswhereArgs | Out-String
+    try {
+        $installations = $vswhereOut | ConvertFrom-Json
+    }
+    catch {
+        Write-Host -f Red "vswhere output = $vswhereOut"
+        throw $_
+    }
```

Comment:
Curious (only): What changed that requires this `try` / `catch` now?

Reply:
Nothing changed. I added this to improve console output when something goes wrong. I was making testing changes to the vswhere code which had a bug and the error was hidden. I added this so I could see the problem, and figured it might be generally useful for the future.